README.md

```diff
@@ -12,17 +12,37 @@ This is the source code of the [web site](http://fr.openfisca.org) for the commu
 
 ## Install
 
+> This project uses [Next.js](https://github.com/zeit/next.js) and [React](https://reactjs.org).  
+
+To install this project's dependencies, go to its directory and run:
 ```sh
-yarn init (creates package.json)
-yarn add next
-yarn add react
-yarn add react-dom
-Create pages/index.js
-add script "dev": "next" in the package.json
-yarn install
-yarn run dev
+npm install
```

Comment:
I think npm run build needs to be added here

Reply:
Testé et a priori nécessaire uniquement pour npm start.